x-pack/plugins/siem/public/components/open_timeline/index.test.tsx

```diff
@@ -4,38 +4,61 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { cloneDeep, get } from 'lodash/fp';
-import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { mount, ReactWrapper } from 'enzyme';
+import { get } from 'lodash/fp';
+import { MockedProvider } from 'react-apollo/test-utils';
 import * as React from 'react';
 
+import { wait } from '../../lib/helpers';
+import { TestProviderWithoutDragAndDrop, apolloClient } from '../../mock/test_providers';
+import { mockOpenTimelineQueryResults } from '../../mock/timeline_results';
 import { DEFAULT_SEARCH_RESULTS_PER_PAGE } from '../../pages/timelines/timelines_page';
-import { mockTimelineResults } from '../../mock/timeline_results';
+
+import { StatefulOpenTimeline } from '.';
 import { NotePreviews } from './note_previews';
 import { OPEN_TIMELINE_CLASS_NAME } from './helpers';
-import { StatefulOpenTimeline } from '.';
-import { TimelineResult } from './types';
 
-describe('StatefulOpenTimeline', () => {
-  const title = 'All Timelines / Open Timelines';
+const getStateChildComponent = (
+  wrapper: ReactWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>
+): React.Component<{}, {}, any> =>
+  wrapper
+    .childAt(0)
+    .childAt(0)
+    .childAt(0)
+    .childAt(0)
+    .childAt(0)
+    .childAt(0)
+    .childAt(0)
+    .childAt(0)
+    .instance();
```

Comment:
Wow. I take it no data test subjects can be used here. No changes requested, just mentioning I don't know which component this is getting.